oxygen-addon.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xt:extensions xmlns:xi="http://www.w3.org/2001/XInclude"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:xt="http://www.oxygenxml.com/ns/extension"
+	xsi:schemaLocation="http://www.oxygenxml.com/ns/extension http://www.oxygenxml.com/ns/extension/extensions.xsd">
+
+	<!-- This @id value assumes that https://github.com/xspec/ community owns "xspec.io". -->
+	<xt:extension id="io.xspec.xspec">
+
+		<!-- To publish a new version, update this @href to a specific commit archive and increment
+			<xt:version>. -->
+		<xt:location
+			href="https://github.com/xspec/xspec/archive/b19ccd7e181fddc21bf191ce27d67cefd65bcd1f.zip" />
+
+		<!-- The version of this add-on. Increment only the last numeric part ('z' of "0.0.z")
+			whenever we publish a new version of this add-on.
+			Ideally, this version number should be the same as the version of XSpec being installed
+			by this add-on. But <xt:version> consists of only three numeric parts ("x.y.z") which
+			doesn't accommodate pre-release versions of XSpec ("x.y.z-SNAPSHOT.1" or something like
+			that). Use "0.0.z" until we find a better way. -->
+		<xt:version>0.0.2</xt:version>
+
+		<!-- Note that supporting multiple oXygen versions may be hard to maintain, because
+			* oXygen add-on and framework require manual testing.
+			* oXygen framework cannot always be backward compatible. See
+			  https://github.com/TEIC/oxygen-tei/issues/30. -->
+		<xt:oxy_version>19.0+</xt:oxy_version>
+
+		<xt:type>framework</xt:type>
+
+		<!-- This could be Jeni, but she's not participated in this add-on release. -->
+		<xt:author>https://github.com/xspec/xspec</xt:author>
+
+		<xt:name>XSpec</xt:name>
+
+		<xt:description>
+			<xi:include href="editors/oxygen/add-on/description/latest.xhtml">
+				<xi:fallback>Visit https://github.com/xspec/xspec and see
+					editors/oxygen/add-on/description/latest.xhtml</xi:fallback>
+			</xi:include>
+		</xt:description>
+
+		<xt:license>
+			<xi:include href="LICENSE" parse="text">
+				<xi:fallback>Visit https://github.com/xspec/xspec and see LICENSE</xi:fallback>
+			</xi:include>
+		</xt:license>
+
+	</xt:extension>
+
+	<!-- This <xt:extension> element is only for upgrade testing. Remove this element later. -->
```

Comment:
If the 2nd x:extension element is removed, does the 0-0-1.xhtml file go away, too?

Reply:
Yes, I'm thinking to do this procedure:
1. First, publish v0.0.2 (and mostly hidden v0.0.1) (as implemented in this pull request)
1. Next, publish v0.0.3 and retire all the previous versions (v0.0.2 and v0.0.1)
   * update `<xt:location>` (to a certain commit) and `<xt:version>` (to `0.0.3`) in the 1st `<xt:extension>`
   * update `latest.xhtml`
   * remove the 2nd `<xt:extionsion>`
   * remove `0-0-1.xhtml`
1. Later on, publish v0.0.4 (and repeat this step for v0.0.5, v0.0.6...)
   * update `<xt:location>` and `<xt:version>` in the 1st and only `<xt:extension>`
   * update `latest.xhtml`

Reply:
Sounds good.